<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Style Sheet</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QFrame" name="mainFrame" >
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="6" column="0" colspan="5" >
        <widget class="QCheckBox" name="agreeCheckBox" >
         <property name="toolTip" >
          <string>Please read the LICENSE file before checking</string>
         </property>
         <property name="text" >
          <string>I accept the terms and &amp;conditions</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Profession:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
         </property>
         <property name="buddy" >
          <cstring>professionList</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="nameLabel" >
         <property name="text" >
          <string>&amp;Name:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring>nameCombo</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QRadioButton" name="maleRadioButton" >
         <property name="toolTip" >
          <string>Check this if you are male</string>
         </property>
         <property name="text" >
          <string>&amp;Male</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="passwordLabel" >
         <property name="text" >
          <string>&amp;Password:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring>passwordEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="4" >
        <widget class="QComboBox" name="countryCombo" >
         <property name="toolTip" >
          <string>Specify country of origin</string>
         </property>
         <property name="statusTip" >
          <string>Specify country of origin</string>
         </property>
         <property name="currentIndex" >
          <number>6</number>
         </property>
         <item>
          <property name="text" >
           <string>Egypt</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>France</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Germany</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>India</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Italy</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Norway</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Pakistan</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="ageLabel" >
         <property name="text" >
          <string>&amp;Age:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring>ageSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="0" >
        <widget class="QLabel" name="countryLabel" >
         <property name="text" >
          <string>Country:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring>countryCombo</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="genderLabel" >
         <property name="text" >
          <string>Gender:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="4" >
        <widget class="QLineEdit" name="passwordEdit" >
         <property name="toolTip" >
          <string>Specify your password</string>
         </property>
         <property name="statusTip" >
          <string>Specify your password</string>
         </property>
         <property name="text" >
          <string>Password</string>
         </property>
         <property name="echoMode" >
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="2" >
        <widget class="QRadioButton" name="femaleRadioButton" >
         <property name="toolTip">
          <string>Check this if you are female</string>
         </property>
         <property name="text" >
          <string>&amp;Female</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2" >
        <widget class="QSpinBox" name="ageSpinBox" >
         <property name="toolTip" >
          <string>Specify your age</string>
         </property>
         <property name="statusTip" >
          <string>Specify your age</string>
         </property>
         <property name="minimum" >
          <number>12</number>
         </property>
         <property name="value" >
          <number>22</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="4" >
        <widget class="QComboBox" name="nameCombo" >
         <property name="toolTip" >
          <string>Specify your name</string>
         </property>
         <property name="editable" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="4" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="3" colspan="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>61</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="3" colspan="2" >
        <widget class="QDialogButtonBox" name="buttonBox" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons" >
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="4" >
        <widget class="QListWidget" name="professionList" >
         <property name="toolTip" >
          <string>Select your profession</string>
         </property>
         <property name="statusTip" >
          <string>Specify your name here</string>
         </property>
         <property name="whatsThis" >
          <string>Specify your name here</string>
         </property>
         <property name="currentRow" >
          <number>0</number>
         </property>
         <item>
          <property name="text" >
           <string>Developer</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Student</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Fisherman</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="editStyleAction" />
    <addaction name="separator" />
    <addaction name="exitAction" />
   </widget>
   <widget class="QMenu" name="menu_Help" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="aboutAction" />
    <addaction name="aboutQtAction" />
   </widget>
   <addaction name="menu_File" />
   <addaction name="menu_Help" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="exitAction" >
   <property name="text" >
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="aboutQtAction" >
   <property name="text" >
    <string>About Qt</string>
   </property>
  </action>
  <action name="editStyleAction" >
   <property name="text" >
    <string>Edit &amp;Style...</string>
   </property>
  </action>
  <action name="aboutAction" >
   <property name="text" >
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
